Content.Server/Flash/DestroyedByFlashingSystem.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Shared.Flash;
+using Content.Server.Flash;
+
+namespace Content.Server.Spreader;
+
+public sealed class DestroyedByFlashingSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<FlashableComponent, FlashAttemptEvent>(OnFlashAttempt);
```

Comment:
change to DestroyedByFlashingComponent

also should this be using an attempt event

Reply:
doesn't work at another event.

Reply:
ok then